DEVELOPER_GUIDE.md

```diff
@@ -274,15 +274,12 @@ curl localhost:9200
 
 Additionally, it is also possible to run a cluster with security enabled:
 ```shell script
-./gradlew run -Dsecurity.enabled=true -Dhttps=true -Duser=admin -Dpassword=admin
+./gradlew run -Dsecurity.enabled=true -Dhttps=true -Duser=admin -Dpassword=myStrongPassword123!
```

Comment:
this should be `<admin-password>`. 

`myStrongPassword123!` is **not** the default password. In fact there is not default password for admin starting 2.12. A custom password needs to be supplied when using demo configuration admin user.